README.md

```diff
@@ -288,6 +289,20 @@ we are getting this output:
       },
       "required":false
     },
+    "noParameterName":{
+      "flowType":{
+        "name":"signature",
+        "type":"function",
+        "raw":"string => void",
+        "signature":{
+          "arguments":[
+            { "name":"_", "type":{ "name":"string" } }
```

Comment:
What do you think about instead of using `'_'`, setting the field to `null`? It seems clearer to me that a consumer of react-docgen checks if the `name === null` instead of a "random" string `name === '_'`

Reply:
Hi @danez,

Got the inspiration from a flow-addon for atom (see screenshot). But I agree that `null` would be a better and cleaner value for working with the data afterwards. Will update the PR.

![screenshot from 2017-10-05 22-16-02](https://user-images.githubusercontent.com/1467188/31250462-217bdffc-aa1b-11e7-8e99-fa609d646361.png)

Reply:
Edit: `getNameOrValue` returns a string value, and therefore `null` would break flow. What do you think @danez?

Reply:
Hmm yes seems it is used quite a bit. We could maybe return an empty string? Then `!name` would work in the resulting documentation.
My intention is just that we should return something else than a non-empty string, because in the `'_'` case you would not be able to differ if the variable name was `_` or empty.

Reply:
Updated the PR setting the name to an empty string now.